core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -1962,7 +1962,9 @@ class KafkaApisTest {
         ArgumentMatchers.eq(producerId),
         ArgumentMatchers.eq(epoch),
         ArgumentMatchers.eq(Set(new TopicPartition(Topic.GROUP_METADATA_TOPIC_NAME, partition))),
+        ArgumentMatchers.eq(false),
```

Comment:
It would be great if we could also add new unit tests to cover the batch mode in `KafkaApisTest`.

Reply:
I added tests in RequestServerTest but I guess those are more integration tests. I can add some in KafkaApisTest too.